packages/@uppy/core/src/Uppy.ts

```diff
@@ -2120,12 +2130,28 @@ export class Uppy<
         })
 
         const { fileIDs } = currentUpload
+        const files = this.getFilesByIds(fileIDs)
+
+        if (this.#uploaders.has(fn)) {
+          const uploaderPlugin = this.#uploaders.get(fn)
+          if (uploaderPlugin) {
```

Comment:
In what condition would this be falsy?

Reply:
any possible edge case where `installingPlugin` is not set. `this.#uploaders.set(fn, this.#installingPlugin ?? '')`

Reply:
I was wondering if you had one (or more) specific case in mind, it's certainly not a blocking concern.